src/features/controls.ts

```diff
@@ -437,42 +434,16 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
     }
 
     /**
-     * Set the camera's radius and field of view to properly frame the scene
-     * based on changes to the model or aspect ratio, and maintains the
-     * relative camera zoom state.
+     * Updates the camera's near and far planes to enclose the scene when
+     * orbiting at the supplied radius.
      */
-    [$updateCamera]() {
-      const controls = this[$controls];
-      const {aspect} = this[$scene];
-      const {idealCameraDistance, fieldOfViewAspect} = this[$scene].model;
-
-      const maximumRadius = idealCameraDistance * 2;
-      controls.applyOptions({maximumRadius});
+    [$setRadius](radius: number) {
```

Comment:
The reason I didn't do this is because it's also dependent on the model's idealCameraDistance, which SmoothControls doesn't know about. This is how I can get the proper far plane when the radius is small or zero (for inside-out mode, see khronos-Sponza and khronos-VC).